.github/workflows/run-tests.yml

```diff
@@ -0,0 +1,16 @@
+name: Run Tests
+
+on: [push]
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v1
+        with:
+          node-version: 12.x
```

Comment:
Let's use node v16 since that is what production will run on

Reply:
If you tell me which specific version of `node` and `npm` you're on, I can work on #320 next with those set.

Reply:
Like I said lets use 16.x + the latest version of npm that goes with it 👍